From 515fb8d20cde7a4b8179ca45647edc4607a5c35b Mon Sep 17 00:00:00 2001
From: Evgeni Golov <evgeni@debian.org>
Date: Wed, 13 Apr 2016 18:04:19 +0200
Subject: [PATCH] do not set the root password in the debian template

closes #302

Signed-off-by: Evgeni Golov <evgeni@debian.org>
---
 templates/lxc-debian.in | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/templates/lxc-debian.in b/templates/lxc-debian.in
index 1ed6f20..b5af844 100644
--- a/templates/lxc-debian.in
+++ b/templates/lxc-debian.in
@@ -140,9 +140,6 @@ EOF
         echo "Timezone in container is not configured. Adjust it manually."
     fi
 
-    echo "root:root" | chroot "$rootfs" chpasswd
-    echo "Root password is 'root', please change !"
-
     return 0
 }
 
